fuse/data/tokenizers/modular_tokenizer/inject_utils.py

```diff
@@ -113,7 +120,11 @@ def build_placeholder_meta_tokenization(
                     raise Exception(f"tokenizer_type={tokenizer_type} is not supported")
 
                 with_placeholders.append(seq)
-
+            elif tokenizer_type.startswith("EXTERNAL_EMBEDDINGS_"):
+                with_placeholders.append(
+                    "<@TOKENIZER-TYPE=AA>"
```

Comment:
Lets maintain a default subtokenizer and use it instead of AA.
We can't assume there is a AA subtokenizer.
(I know that such an assumption was in scalars implementation also)

Reply:
done